src/main/java/complete_task/completeTaskController.java

```diff
@@ -1,20 +1,18 @@
 package complete_task;
-import entity_layer.Curriculum;
 
 /**
  * Controller for the completeTask use case; receives user input in
- * the form of the id of the task to be completed and the curriculum
- * that the task belongs to.
+ * the form of the id of the task to be completed, the id of the
+ * curriculum that the task belongs to and the active user.
  *
  * @author Bmguiler
  */
 public class completeTaskController {
     public static int taskId;
-    public static Curriculum taskCurriculum;
```

Comment:
These attributes should be private and not static. In Java, it is always best to make variables as 'protected' as possible. The controller should use the interactor, but I'm not seeing this interaction in the use case